Puppetfile

```diff
@@ -1,12 +1,23 @@
-forge 'https://forgeapi.puppetlabs.com'
+forge 'https://forgeapi.puppetlabs.com/'
 
-mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
-mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
-mod 'katello/pulp',                  :git => 'https://github.com/theforeman/puppet-pulp'
-mod 'katello/qpid',                  :git => 'https://github.com/theforeman/puppet-qpid'
-
-# A fork that marks metadata as compatible with puppet 5
-# https://github.com/thias/puppet-squid3/pull/59
+mod 'katello/candlepin',             '>= 5.2.1 < 5.3.0'
+mod 'katello/foreman_proxy_content', '>= 7.3.0 < 7.4.0'
+mod 'katello/certs',                 '>= 4.3.0 < 4.4.0'
+mod 'katello/katello',               '>= 8.2.0 < 8.3.0'
+mod 'katello/pulp',                  '>= 5.6.0 < 5.7.0'
+mod 'katello/qpid',                  '>= 4.3.0 < 4.4.0'
 mod 'thias-squid3',                  :git => 'https://github.com/theforeman/puppet-squid3', :ref => 'allow-puppet-5'
+
+# Dependencies
```

Comment:
This was copied in from Foreman's Puppetfile. In the past we always removed these so they're only maintained in a single place.